Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIsoTrkProducerFilter_AlCaRaw_cff.py

```diff
@@ -0,0 +1,20 @@
+import FWCore.ParameterSet.Config as cms
+
+#----------------------------------------------------
+# AlCaReco filtering for running on dedicated AlCaRaw
+#----------------------------------------------------
+
+from Calibration.HcalAlCaRecoProducers.alcaHcalIsotrkProducer_cff import *
+from Calibration.HcalAlCaRecoProducers.alcaHcalIsotrkFilter_cfi import *
+from RecoLocalCalo.EcalRecAlgos.EcalSeverityLevelESProducer_cfi import *
+
+alcaHcalIsotrkProducer.labelTrack("hltMergedTracksSelector")
+alcaHcalIsotrkProducer.labelVertex("hltTrimmedPixelVertices")
+alcaHcalIsotrkProducer.labelEBRecHit(edm.InputTag("hltEcalRecHit", "EcalRecHitsEB", ""))
+alcaHcalIsotrkProducer.labelEERecHit(edm.InputTag("hltEcalRecHit", "EcalRecHitsEE", ""))
+alcaHcalIsotrkProducer.labelHBHERecHit("hltHbhereco")
+alcaHcalIsotrkProducer.labelBeamSpot("hltOnlineBeamSpot")
+alcaHcalIsotrkProducer.labelMuon("hltIterL3Muons")
+alcaHcalIsotrkProducer.labelTriggerEvent(edm.InputTag("triggerTriggerEvent", "hltTriggerSummaryAOD", "HLT"))
+
+seqALCARECOHcalCalIsoTrkProducerFilter = cms.Sequence(alcaHcalIsotrkProducer * alcaHcalIsotrkFilter)
```

Comment:
out of curiosity, what's the purpose of this file?

Reply:
Hi Marco,

It is my understanding that the similarly named cff file (without the `AlCaRaw` in the name) had been used for configuration of the `HcalCalIsoTrkProducerFilter` ALCARECO. Thus, this new version here would be for configuring the same ALCARECO for running on the new `AlCaHcalIsoTrk` data stream, where the associated `AlCaHcalIsotrkProducer` producer needs to be pointed to the HLT-specific collections in the stream.

Reply:
Hi @JHiltbrand, how do you foresee the configuration builder to pick up the content of this file if it's a new file and not referenced elsewhere in cmssw?

Reply:
Ahh, I see 🤦‍♂️ will get properly referenced